### PR TITLE
Update styles for docs

### PR DESCRIPTION
**Pull request by IsAmrish**
This PR will
- use `scss module` instead of `css module`
- change css directory to scss
- define the breakpoints
- define the colors and colors from css root

Fixes: #108

### CODE REVIEW

**Review by rakeshPRaghu — APPROVED** [review]
lgtm 👍🏼

File: docs/src/components/ImgCard/index.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 function ImgCard({ dataList }) {
   return (
-    <div className="img-wrapper">
+    <div className={styles.wrapper}>
       {dataList.map((data) => {
         return (
-          <Link className="img-block" to={data?.url} key={data?.title}>
-            <img src={data?.Svg} />
+          <Link to={data?.url} key={data?.title} className={styles.link}>
+            <img src={data?.Svg} className={styles.img} />
```
Comment (Pallavi-PH) [inline docs/src/components/ImgCard/index.js:11]: We use `classes` instead of `styles` on the website. Can we use the same for consistency?

Comment (Pallavi-PH) [reply docs/src/components/ImgCard/index.js:11]: Same in other places as well

**Review by satyapriyamishra222 — APPROVED** [review]

**Review by AVRahul — APPROVED** [review]

File: docs/src/components/ImgCard/index.js (hunk 1 above)
Comment (IsAmrish) [reply docs/src/components/ImgCard/index.js:11]: Website is using `material-ui`  where normal conventions is to use `classes`. Docs code is totally based upon `docusaurus` where we haven't used the material ui instead used mix of `scss` and `scss-module` . Normally `css-module`'s convention is to use `styles` ( as name ).

I am agree that we are using single repo for both the website source code but still both the code are different ( one is based on react, typescript and material ui && second one is based upon docusaurus, react, javascript and scss, scss-in-module )

Also we are not using the concept of mono-repo ( where some components would have been common in both website )

Comment (Pallavi-PH) [reply docs/src/components/ImgCard/index.js:11]: Ok

**Review by Pallavi-PH — APPROVED** [review]

**Review by satyapriyamishra222 — APPROVED** [review]

**Pull request merged**